P2CODE/AA1/UWBAnchor/ADOPTERS.yaml

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model Ultra Wide Band UWBAnchor of the Subject dataModel.Aeronautics.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```

Comment:
all fields non mandatory can be removed., Possibly I'll do after merging in incubated

Reply:
done